setup.cfg

```diff
@@ -45,12 +45,3 @@ exclude =
 
 [bdist_wheel]
 universal = 1
-
-[flake8]
-max-complexity = 10
-max-line-length = 119
-exclude = .venv,djstripe/migrations/,.tox
-
-[pycodestyle]
-max-line-length = 119
-exclude = .venv,djstripe/migrations/,.tox
```

Comment:
I moved it to tox.ini where it belongs. I know it can be in both but it's more common to think of `tox.ini` as "the place for test related settings" and `setup.cfg` as "the place for setup-related settings".

Tox unfortunately doesn't support being in setup.cfg so we have to have two files...